.gitattributes

```diff
@@ -1 +1 @@
-* text=auto
+* text eol=lf
```

Comment:
Can't do that. It makes even binary be text.

The correct way is to set explicit file extensions to be `lf`.